lib/r10k/forge/module_release.rb

```diff
@@ -205,6 +270,13 @@ def cleanup_md5_file_path
           md5_file_path.delete
         end
       end
+
+      # Remove the module release sha256.
+      def cleanup_sha256_file_path
```

Comment:
This convention is wild.

Reply:
What do you mean, specifically?

Reply:
the abstraction over a call to `#delete`, and an existence check that most callers will already know the answer to.

not your code, but i dislike the conventions around the `cleanup_*_path` methods, and prefer methods that are _not_ part of an imperative, high level, public api to only act on their explicit inputs.

Reply:
I can ... cleanup... that too ;)